projects/packages/videopress/src/client/block-editor/blocks/video/edit.tsx

```diff
@@ -144,52 +144,8 @@ export default function VideoPressEdit( {
 	const token = useMediaToken( { guid, isPrivate, tokenType: 'playback' } );
 
 	// Store if the chapters file is auto-generated in a local state.
-	const [ isAutoGeneratedChapter, setIsAutoGeneratedChapter ] = useState( true );
-
-	// Detect if the chapter file is auto-generated.
```

Comment:
This comment is missing in the changes, could we incorporate it along with its associated line?